### ISSUE TITLE
Search function

### ISSUE THREAD

**Issue by laraveliouser** [mrakodol/Laravel-5-Bootstrap-3-Starter-Site#96]
On the admin panel, tried search bar in news. Datatables tries to run a query using the full column name with the dot, backtickked as a whole. So \`articles.id\` causes an error, it probably needs to be \`articles\`.\`id\` instead.
Has this functionality worked before? Is it recently broken? The same error can be seen on the demo site.

**Comment by roryza** [comment]
I think I might be experiencing the same issue except I didn't spot the error but recently the search bar stopped working but it was after I updated modules etc. I will compare queries with a working version.

**Comment by DeadDuck** [comment]
Ok it's a problem with the DataTables package v 4.1.4. The search query was shaped as below:
```sql
select
  count(*) as aggregate
from (
  select
    '1' as row_count
  from `articles`
    join `languages` on `languages`.`id` = `articles`.`language_id`
    join `article_categories` on `article_categories`.`id` = `articles`.`article_category_id`
  where
    (
    LOWER(`articles.id`) LIKE '%a%'
    or LOWER(`articles.title`) LIKE '%a%'
    or LOWER(`languages.name`) LIKE '%a%'
    or LOWER(`articles.created_at`) LIKE '%a%'
    )
  order by
    `articles`.`position` asc
  ) count_row_table
```
In the `LOWER` parameters, column names are wrongly escaped by the `wrapColumn` function in the DataTables package. Apparently this function was fixed in V4.1.5 three days ago.

**Comment by stojankukrika** [comment]
I update datatable to 4.2
